predictor_robojudge.ipynb

```diff
@@ -497,36 +497,27 @@
     "# Ranking"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Global"
-   ]
-  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
```

Comment:
Instead of recomputing the cumulative value every time, we now just grab it's value at the last day.